package.json

```
@@ -56,6 +65,7 @@
     "esbuild": "^0.19.10",
     "eslint": "^8.56.0",
     "eslint-plugin-jsdoc": "^48.0.4",
-    "mocha": "^10.2.0"
+    "mocha": "^10.2.0",
+    "typescript": "^5.3.3"
```

What do you think about changing this to a tilde since TypeScript doesn't follow semver?

```suggestion
    "typescript": "~5.3.3"
```

https://github.com/microsoft/TypeScript/issues/14116